platform/pulp/platform/models/progress.py

```diff
@@ -19,6 +19,19 @@
 _logger = logging.getLogger(__name__)
 
 
+def get_current_task_id():
```

Comment:
I copied this this function out of pulp.server.async.tasks, but I don't know that it should live here. @bmbouter?